pkg/status/versionchecks_test.go

```diff
@@ -0,0 +1,118 @@
+package status
+
+import (
+	"net/http"
+	"testing"
+	"time"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	"github.com/codeready-toolchain/toolchain-common/pkg/test"
+	"github.com/google/go-github/v52/github"
+	"github.com/migueleliasweb/go-github-mock/src/mock"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func TestCheckDeployedVersionIsUpToDate(t *testing.T) {
```

Comment:
could you please add the `// given`, `// when`, and `// then` comments to each section of the sub-tests? It's easier to read

Reply:
sure, let me fix this!

Reply:
done, please see: https://github.com/codeready-toolchain/toolchain-common/pull/315/commits/40c65b28c47481b17a33888c6e1fedb1c60c2bad

Reply:
I missed few places in the previous commit, please check also: https://github.com/codeready-toolchain/toolchain-common/pull/315/commits/dd6a8284e2c66ee824a14421480d97d3a92278dd